src/analysis/inner_product_space/spectrum.lean

```diff
@@ -244,3 +244,23 @@ end version2
 
 end is_self_adjoint
 end inner_product_space
+
+section nonneg
+
```

Comment:
I think we might as well have this `simp` lemma. See below.
```suggestion
@[simp]
lemma inner_product_apply_eigenvector {μ : 𝕜} {v : E} {T : E →ₗ[𝕜] E}
  (h : v ∈ module.End.eigenspace T μ) : ⟪v, T v⟫ = μ * ∥v∥^2 :=
by simp only [mem_eigenspace_iff.mp h, inner_smul_right, inner_self_eq_norm_sq_to_K],
```

Reply:
I am pro-`simp` lemmas in general.  How careful does one need to be when adding these?